tokio/src/sync/watch.rs

```diff
@@ -10,24 +10,64 @@
 //!
 //! [`channel`] returns a [`Sender`] / [`Receiver`] pair. These are the producer
 //! and consumer halves of the channel. The channel is created with an initial
-//! value. The **latest** value stored in the channel is accessed with
-//! [`Receiver::borrow()`]. Awaiting [`Receiver::changed()`] waits for a new
-//! value to be sent by the [`Sender`] half.
+//! value.
+//!
+//! Each [`Receiver`] independently tracks the last value *seen* by its caller.
+//!
+//! To access the **latest** value stored in the channel and mark it as *seen*,
+//! use [`Receiver::borrow_and_update()`].
```

Comment:
do you think it's worth explicitly stating something like "`borrow_and_update()` should be preferred if the receiver intends to await notifications from `changed()`"?

Reply:
Moved it into a new section titled "`borrow_and_update` versus `borrow`". Hopefully that works.